changes/tests.py

```diff
@@ -3593,10 +3593,50 @@ def test_add_change_unsubscribe_household(self):
         self.assertEqual(len(responses.calls), 3)
 
     @responses.activate
+    def test_add_change_unsubscribe_household_no_msisdn(self):
+        # Setup
+        self.make_source_adminuser()
+        mother_id = "4038a518-2940-4b15-9c5c-2b7b123b8735"
+        household_id = "4038a518-2940-4b15-9c5c-9ix9cvx09cv8"
+
+        details = {
+            'linked_to': mother_id
+        }
+
+        self.mock_identity_lookup("%2B2347031221928", household_id, details)
+        self.mock_identity_optout()
+
+        post_data = {
+            "action": "unsubscribe_household_only",
+            "data": {
+                "reason": "not_useful",
+                "household_msisdn": "07031221928"
+            }
+        }
+        # Execute
+        response = self.adminclient.post('/api/v1/addchange/',
+                                         json.dumps(post_data),
+                                         content_type='application/json')
+
+        # Check
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        d = Change.objects.last()
+        self.assertEqual(d.source.name, 'test_ussd_source_adminuser')
+        self.assertEqual(d.action, 'unsubscribe_household_only')
+        self.assertEqual(d.validated, False)
```

Comment:
There is no code that validates changes at the moment. All changes in Production are validated=False.

I have spoken to Chima about this and there should be a ticket for validation soon.